src/index.js

```diff
@@ -0,0 +1,108 @@
+import './style.css';
+import Icon from './option.png';
+import validateForm from './funct.js';
+import removeTask from './remove.js';
+import removeAllTask from './removeAll.js';
+
+let tasks = [];
+export const main = document.querySelector('.container');
+const dataLoading = () => {
+  tasks = JSON.parse(localStorage.getItem('datas')) ?? [];
+  main.innerHTML = '';
+  /* eslint-disable */
+  document.body.appendChild(component());
+  /* eslint-enable */
+};
+
+export default dataLoading;
+const showTask = (i) => {
+  const li = document.createElement('li');
+  const inputCheckbox = document.createElement('input');
+  const paragraph = document.createElement('input');
+  inputCheckbox.setAttribute('type', 'checkbox');
+  if (tasks[i].completed === false) {
```

Comment:
Since it´s a boolean condition, you could shortcut here to `!tasks[i].completed`

Reply:
Thank you for this feedback, I didn't know if it was possible.